java/build-android.gradle

```diff
@@ -1,9 +1,27 @@
 apply plugin: 'com.android.library'
+apply plugin: 'maven-publish'
 
 def jniLibsDir = System.properties['jniLibsDir']
 def buildDir = System.properties['buildDir']
+def publishDir = System.properties['publishDir']
+
+// Since Android requires a higher numbers indicating more recent versions
+// This function assume ORT version number will be in formart of A.B.C such as 1.7.0
+// We generate version code A[0{0,1}]B[0{0,1}]C,
+// for example '1.7.0' -> 10700, '1.6.15' -> 10615
+def getVersionCode(String version){
+	String[] codes = version.split('\\.');
```

Comment:
Do we have plan to publish nightly package? If so this may not be a good way for version number, since all version "1.7.0" will have 10700 version code

Reply:
I would expect we'd either need a different package name, or an extra segment  (e.g. 1.7.0.x) for a nightly. otherwise wouldn't it clash with say a 1.7.1 patch release?

no plan to have a nightly though

---
In reply to: [602553604](https://github.com/microsoft/onnxruntime/pull/7138#discussion_r602553604) [](ancestors = 602553604)

Reply:
I think we can stick with this for now, any worry it later if we plan to release the nightly package